README.md

```diff
@@ -1,7 +1,11 @@
 # delaunay-triangulation
 
+[![Build Status](https://travis-ci.org/Gjacquenot/delaunay-triangulation.svg?branch=master)](https://travis-ci.org/Gjacquenot/delaunay-triangulation)
```

Comment:
Does this needs to be changed if merged into master ?

Reply:
Yes, once merged you need to create a travis-ci account and update the address to match your account.